### ISSUE TITLE
ErrorException: Notice: Constant GK_SITE_BASE_SERVER_URL already defined

### ISSUE THREAD

**Issue by kumy** [geokrety/geokrety-website#507]
https://sentry.kumy.org/geokrety/geokrety-legacy/issues/5308/?referrer=github_plugin

```
ErrorException: Notice: Constant GK_SITE_BASE_SERVER_URL already defined
  File "/app/GeoKrety/Service/Config.php", line 8, in GeoKrety\Service\Config::__construct
    define('GK_SITE_BASE_SERVER_URL', $_ENV['GK_SITE_BASE_SERVER_URL'] ?? 'https://geokrety.org');
  File "/app/GeoKrety/Controller/HealthCheck.php", line 20, in GeoKrety\Controller\HealthCheck::checkWebsiteConfig
    $websiteConfig = new \GeoKrety\Service\Config();
  File "/app/GeoKrety/Controller/HealthCheck.php", line 13, in GeoKrety\Controller\HealthCheck::get
    $this->checkWebsiteConfig();
  File "/vendor/bcosca/fatfree-core/base.php", line 1873, in Base::call
    $out=call_user_func_array($func,$args?:[]);
  File "/vendor/bcosca/fatfree-core/base.php", line 1673, in Base::run
    'beforeroute,afterroute');
...
(1 additional frame(s) were not displayed)
```

**Comment by kumy** [comment]
Relates to #500.

**Comment by boly38** [comment]
One thing I have to understand is why I didn't produced this error locally (because I have made some tests with /health endpoint)

**Comment by boly38** [comment]
With or withour your 507 fix, I'm unable to reproduce this error.

With local compose, if I sh to the web container I checked that I've got this environment variable `GK_SITE_BASE_SERVER_URL`.

Why in sentry use case we have this constructor called twice ? 

I only found one instance of the constructor in `index.php`::l10

**Comment by kumy** [comment]
No matter if sentry is enabled or not, when calling `/health`, both constructor are called, as the `index.php` is always executed.

Probably an `error_reporting` level issue?

**Comment by kumy** [comment]
Found this in apache-php container `/usr/local/etc/php/php.ini-production` L462

```
; Common Values:
;   E_ALL (Show all errors, warnings and notices including coding standards.)
;   E_ALL & ~E_NOTICE  (Show all errors, except for notices)
;   E_ALL & ~E_NOTICE & ~E_STRICT  (Show all errors, except for notices and coding standards warnings.)
;   E_COMPILE_ERROR|E_RECOVERABLE_ERROR|E_ERROR|E_CORE_ERROR  (Show only errors)
; Default Value: E_ALL & ~E_NOTICE & ~E_STRICT & ~E_DEPRECATED
; Development Value: E_ALL
; Production Value: E_ALL & ~E_DEPRECATED & ~E_STRICT
; http://php.net/error-reporting
error_reporting = E_ALL & ~E_DEPRECATED & ~E_STRICT
```

**Comment by boly38** [comment]
Yep index is always used
but even with this in apache vhost config checked with phpinfo), I've no warning 

    php_admin_flag display_errors on
    php_admin_value error_reporting 30719

**Comment by kumy** [comment]
The framework updates the `error_reporting` level itself

https://github.com/bcosca/fatfree/blob/9923d7bbd94ccf63f53756543cd974fdd01f06f4/lib/base.php#L2248

@boly38 Please see https://github.com/geokrety/geokrety-website/pull/511/commits/a0771a08b5213992eaf25d6dbee99b8921b60790 included in PR #511 which should allow to display them when `GK_F3_DEBUG` is `true`

**Comment by boly38** [comment]
Excellent thanks 👏
